docs/tutorials/getting-started.md

```diff
@@ -5,16 +5,50 @@ This page will guide you through getting started with PyScript.
 ## Development setup
 
 PyScript does not require any development environment other
-than a web browser. We recommend using [Chrome](https://www.google.com/chrome/).
+than a web browser (we recommend using [Chrome](https://www.google.com/chrome/)) and a text editor, even though using your [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) of choice might be convenient.
 
 If you're using [VSCode](https://code.visualstudio.com/), the
 [Live Server extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
 can be used to reload the page as you edit the HTML file.
 
+## Trying before installing
+
+If you're new to programming and know nothing about HTML or just want to try some of PyScript features, we recommend using the [](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) [PyScript REPL example](https://pyscript.net/examples/repl.html) instead so you can have a programming experience in a REPL that doesn't require any setup. This REPL can be used to have an interactive experience using Python directly.
```

Comment:
>`[](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)`

Is this a typo?

Reply:
Yeah, I meant to link the word "REPL" to Wikipedia, I'll fix it.

Reply:
I see, cool!